brooklyn-tosca-transformer/src/test/resources/templates/mysql-topology-overwrited-interface.tosca.yaml

```diff
@@ -0,0 +1,60 @@
+tosca_definitions_version: tosca_simple_yaml_1_0_0_wd03
+description: Alien4Cloud generated service template
+template_name: basic-compute-with-inputs
+template_version: 0.1.0-SNAPSHOT
+template_author: admin
+
+imports:
+  - mysql-type:2.0.0-SNAPSHOT
+  - tosca-normative-types:1.0.0.wd06-SNAPSHOT
+  - brooklyn-types:0.1.0-SNAPSHOT
+
+topology_template:
+  node_templates:
+    Compute:
+      name: Compute
+      type: brooklyn.nodes.Compute
+      capabilities:
+        endpoint:
+          properties:
+            protocol: tcp
+            initiator: source
+            network_name: PRIVATE
+            secure: true
+        scalable:
+          properties:
+            min_instances: 1
+            max_instances: 1
+            default_instances: 1
+    Mysql:
+      name: Mysql
+      type: alien.nodes.Mysql
+      properties:
+        db_port: 3361
+        db_name: wordpress
+        db_user: martin
+        db_password: pass
+        bind_address: true
+        storage_path: /mountedStorage
+        port: 3306
+        name: wordpress
+      requirements:
+        - host:
+            node: Compute
+            capability: tosca.capabilities.Container
+            relationship: tosca.relationships.HostedOn
+      capabilities:
```

Comment:
Almost all of this file is irrelevant to its purpose in a test. Suggest deleting everything extraneous and deleting the irrelevant assertions in `testOverwriteInterfaceOnMysqlTopology`.